dev/devicelab/manifest.yaml

```diff
@@ -148,17 +148,17 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
-  hot_mode_dev_cycle__preview_dart_2_benchmark:
+  hot_mode_dev_cycle__dart1_benchmark:
```

Comment:
nit: let's put "dart1" after "benchmark" (i.e. `__benchmark_dart1`) so that when we need to compare dart1 vs dart2, we can search for a common prefix.